src/univalue/test/unitester.cpp

```diff
@@ -157,6 +157,9 @@ int main (int argc, char *argv[])
 
     unescape_unicode_test();
 
+    char buf[] = "___[1,2,3]___";
+    UniValue val;
+    assert(val.read(buf + 3, 7));
```

Comment:
It might be good to move this into a function, otherwise it will be hard to understand the purpose of the test from looking at the code.

Reply:
Moved into a function.